ssl/record/rec_layer_d1.c

```diff
@@ -665,28 +665,47 @@ int do_dtls1_write(SSL_CONNECTION *sc, uint8_t type, const unsigned char *buf,
     return ret;
 }
 
-void dtls1_increment_epoch(SSL_CONNECTION *s, int rw)
+int dtls1_increment_epoch(SSL_CONNECTION *s, int rw)
```

Comment:
As you are adding a return value, could you please adjust the existing callers to check it?